hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/file/AvroMetadataResolver.java

```diff
@@ -33,7 +33,7 @@
     private static final FileFormat<Map<String, String>> FORMAT = FileFormat.avro();
 
     @Override
-    protected FileFormat<?> format() {
+    protected FileFormat<?> formatForSample() {
```

Comment:
I'm not sure i like `formatForSample` & `formatForData`. Maybe just implement `resolveMetadata` in subclasses?

Reply:
done in 3772e38e1